src/memvid/retriever.py

```
@@ -0,0 +1,179 @@
+            self.fps = cap.get(cv2.CAP_PROP_FPS)
+            cap.release()
+            
+            logger.info(f"Video verified: {self.total_frames} frames at {self.fps} fps")
+        except Exception as e:
+            logger.error(f"Video verification failed: {e}")
+    
+    def search_simple(self, query: str, top_k: int = 5) -> List[str]:
+        """
+        Simple text-based search (fallback when no embeddings available)
+        """
+        query_lower = query.lower()
```

Ruff (0.11.9)

Local variable `query_lower` is assigned to but never used

Remove assignment to unused variable `query_lower`

(F841)